drivers/remote/driver.go

```diff
@@ -229,8 +231,19 @@ func (d *driver) Join(nid, eid string, sboxKey string, jinfo driverapi.JoinInfo,
 			if jinfo.AddStaticRoute(route.Destination, route.RouteType, route.NextHop) != nil {
 				return errorWithRollback(fmt.Sprintf("failed to set static route: %v", route), d.Leave(nid, eid))
 			}
+			if route.Destination.String() == "0.0.0.0/0" {
+				defaultRoute++
+			}
 		}
 	}
+	jinfo.DisableGatewayService(res.DisableGatewayService)
+	if !res.DisableGatewayService {
+		defaultRoute++
+		log.Infof("enabled libnetwork default gateway service")
+	}
+	if defaultRoute > 1 {
```

Comment:
I will let @sanimej & @aboch give their suggestion on a better fix to handle this.